src/arr/compiler/type-check.arr

```diff
@@ -167,27 +167,36 @@ fun type-check(program :: A.Program, compile-env :: C.CompileEnvironment, module
     else:
       uri = globvs.get-value(g)
       # TODO(joe): type-check vars by making them refs
-      context.set-global-types(context.global-types.set(A.s-global(g).key(), compile-env.mods.get-value(uri).values.get-value(g).t))
+
+      if (g == "_"):
```

Comment:
It is odd to me that this is necessary. It feels like something didn't get desugared correctly if we're getting here. But I understand that it's because the global environment is somewhat special, and the only context that deals in raw, un-atomized names.

Reply:
This seems to be a weird interaction between the desugaring of `include` and how the repl treats its code as globals. It is probably worth looking into, but I wanted to get at least a quick (but correct) fix in, so I didn't do too much digging tonight.